src/features/cartography/infrastructure/data/rest/cnfs/cnfs.rest.ts

```diff
@@ -0,0 +1,26 @@
+import type { Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+import type { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import type { Cnfs } from '../../../../core';
+import { CnfsRepository } from '../../../../core';
+import type { CnfsTransfer } from '../../models';
+import { cnfsTransferToCore } from '../../models';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CnfsRest extends CnfsRepository {
+  private readonly _endpointUri: string =
+    'https://MON_API_DOMAINE/api/LIST_CNFS';
```

Comment:
```suggestion
  private readonly _endpointUri: string =
    '$listCnfs';
```
+
Ajouter un intercepteur qui détecte toutes les routes qui commencent par `?` et les remplacent le `$` par `https://MON_API_DOMAINE/api/`, cette url étant stockée dans une configuration (token)

Reply:
#nextPR

Reply:
j'ai rajouté le TODO